### PR TITLE
Add complete task bpmn

### PR DESCRIPTION
**Pull request by lewis150193**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-198

### Change description ###
Added Complete task DMN

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/ia/task/configuration/CamundaSetTaskComplete.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class CamundaSetTaskComplete {
```
Comment (iguannago) [inline src/integrationTest/java/uk/gov/hmcts/reform/ia/task/configuration/CamundaSetTaskComplete.java:19]: could we add the 'Test' in the name of the class? To follow convention.

File: build.gradle [hunk 2]
```diff
@@ -165,8 +189,11 @@ def versions = [
   reformLogging   : '5.1.5',
   springBoot      : springBoot.class.package.implementationVersion,
   springfoxSwagger: '2.9.2',
+  serenity        : '2.2.12',
+  gradlePitest    : '1.5.1',
+  pitest          : '1.5.2',
```
Comment (iguannago) [inline build.gradle:194]: Are we using pitest in this project? If so is it configured in the nightly Jenkins pipeline?

**Review by iguannago — DISMISSED** [review]

File: src/main/resources/complete_task.dmn [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" id="Definitions_05yl6j0" name="DRD" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="4.2.0">
+  <decision id="Decision_0v6a16e" name="Task completion">
+    <decisionTable id="DecisionTable_01re27m" hitPolicy="COLLECT">
+      <input id="eventId" label="Event ID">
+        <inputExpression id="LiteralExpression_1ngg489" typeRef="string">
+          <text>eventId</text>
+        </inputExpression>
+      </input>
+      <output id="Output_1" label="Task Type" name="taskType" typeRef="string" />
+      <output id="OutputClause_1mldd9e" name="Completion" typeRef="string" />
```
Comment (BenNzewi) [inline src/main/resources/complete_task.dmn:11]: Should this label be Completion Mode as per Ticket description?

File: src/integrationTest/java/uk/gov/hmcts/reform/ia/task/configuration/CamundaSetTaskCompleteTest.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        VariableMap result = new VariableMapImpl();
+
+        result.putValue("taskType","List of TaskTypes");
+        result.putValue("Completion","Auto");
+
+        DmnDecisionTableResult dmnDecisionRuleResults = evaluateDmn("anything");
+
+        assertThat(dmnDecisionRuleResults.getFirstResult().getEntryMap(), is(result));
+    }
+
```
Comment (BenNzewi) [inline src/integrationTest/java/uk/gov/hmcts/reform/ia/task/configuration/CamundaSetTaskCompleteTest.java:42]: Do we need a test to capture the edge case when a mapping cannot be matched?

**Review by BenNzewi — APPROVED** [review]
